ask/alexa_io.py

```diff
@@ -50,6 +55,19 @@ def access_token(self):
     def session_id(self):
         return self.request["session"]["sessionId"]
 
+    def application_id(self):
+        try:
```

Comment:
Please replace the generic exception handling with an explicit check, or at least catch the specific exception